app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/event_parser.rb

```diff
@@ -0,0 +1,26 @@
+module ManageIQ::Providers::IbmCloud::VPC::CloudManager::EventParser
+  def self.event_to_hash(event, ems_id)
+    event_hash = {
+      :event_type => event["action"],
+      :source     => "IBM_CLOUD_VPC",
+      :ems_ref    => ems_id,
+      :full_data  => event
+    }
+
+    case event_hash[:event_type]
+    when 'is.instance.instance.create'
+      parse_vm_event!(event, event_hash)
+    end
+
+    event_hash
+  end
+
+  def self.parse_vm_event!(event, event_hash)
+    return if event['responseData'].nil?
+
+    event_hash[:timestamp] =  event["eventTime"]
```

Comment:
We need timestamp in the general event case, can this be moved up [[here]](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/252/files#diff-d705b2bd25d0350d720927209af1307766723f626713b08dd66f14a1b000c4b0R7) or does eventTime only show up for `'is.instance.instance.create'` type events?

Reply:
sure, `eventTime` is available for all events